qml/js/yleApi.js

```diff
@@ -235,16 +308,18 @@ var api = (function(appId, appKey) {
          getProgramsByCategoryId: getProgramsByCategoryId,
          getProgramsBySeriesId: getProgramsBySeriesId,
          search: search,
+         getCurrentRadioBroadcasts: getCurrentRadioBroadcasts,

```

Comment:
This is missing the getNowPlayingRadioPrograms function which is used in FirstPage.qml. Also the function is missing from below in function list. In addition to this there now seems to be two functions which are probably supposed to do the same thing getNowPlayingRadioPrograms and getCurrentRadioBroadcasts.

Reply:
Sorry, renamed the `getNowPlayingRadioPrograms` into `getCurrentRadioBroadcasts` and forgot to rename it into `FirstPage.qml`.

The two methods are using different endpoints, the other (not used atm) returns the currently playing songs/programs in the given radiochannel and which I thought was needed for listening those songs. Not sure if we want to show information about next/current/previous songs somewhere in the app?